src/components/dropdown/dropdown.controller.js

```diff
@@ -132,19 +137,48 @@ export default class ScDropdownCtrl {
     }
 
     _positionVerticalMenu() {
-        const positionAction = this.trigger[0].getBoundingClientRect();
-        const positionMenu = this.content[0].getBoundingClientRect();
-        let menuTopPosition = positionAction.bottom + CARRET_HEIGHT;
+        const blocAction = this.trigger[0];
+        const blocContent = this.content[0];
+        const positionAction = blocAction.getBoundingClientRect();
+        const positionContent = blocContent.getBoundingClientRect();
+        const windowSize = this.window[0].innerHeight;
 
-        //when menu bottom is outside of the window, we position the menu at the top of the button
-        if ((positionAction.bottom +  positionMenu.height + CARRET_HEIGHT)> this.window[0].innerHeight) {
-            menuTopPosition = positionAction.top - CARRET_HEIGHT - positionMenu.height;
-            this.content.addClass('top');
+
+
+
+        //display the menu where there is most space top or bottom
+        const freeSpaceBottom = windowSize - positionAction.bottom;
+        const freeSpaceTop = positionContent.top;
+        const heightContent = positionContent.height;
+        let offsetBorder = parseInt(this.distanceFromBorder);
+
+        if (isNaN(offsetBorder)){
+            offsetBorder = 30;
```

Comment:
offsetBorder should be 0 by default.
This is supposed to be a constant, init it in a $onInit part to avoid doing this check everytime